Add clearer instructions for Auto-open DevTools (#602)
GoogleChrome/developer.chrome.com: site/en/docs/devtools/open/index.md

/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --auto-open-devtools-for-tabs
```

> Will only work if an instance of Chrome is not already running. From then on, every new tab will automatically open DevTools until user fully quits Chrome <kbd>Command</kbd>+<kbd>Q</kbd>.

Reviewer: ```suggestion
This will only work if an instance of Chrome is not already running. From then
on, every new tab will automatically open DevTools until the user fully quits
Chrome.
```

Author: @jpmedley CLA was signed on Apr 04, 2021 00:53 PDT.
